### PR TITLE
test: add ITSystemTest

### PR DESCRIPTION
**Pull request by athakor**
towards #2

### CODE REVIEW

**Comment by athakor** [comment]
@chingor13 PTAL

File: google-cloud-datalabeling/pom.xml [hunk 1]
```diff
@@ -58,6 +58,10 @@
       <groupId>com.google.api</groupId>
       <artifactId>gax-grpc</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-core</artifactId>
+    </dependency>
```
Comment (chingor13) [inline google-cloud-datalabeling/pom.xml]: Please add `<scope>test</scope>` and move it with the other test dependencies

Comment (athakor) [reply google-cloud-datalabeling/pom.xml]: Done

**Comment by athakor** [comment]
@chingor13 PTAL

**Pull request merged**